src/lib/components/layout/v2/AppContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div<{
+  hasPadding?: boolean;
+}>`
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  padding: ${(props) => (props.hasPadding ? '0 1rem' : 'initial')};
```

Comment:
I'm not sure the padding should be applied here because the sidebar should stick to the left without padding.
The padding option should likely better be on the MainContentContainer